lib/immortal.rb

```diff
@@ -93,20 +92,19 @@ def undeleted_clause_sql
     def deleted_clause_sql
       unscoped.where(arel_table[:deleted].eq(true)).constraints.first.to_sql
     end
-
   end
 
   module InstanceMethods
     def self.included(base)
-      unless base.table_exists? && base.columns_hash["deleted"] && !base.columns_hash["deleted"].null
-        Kernel.warn "[Immortal] The 'deleted' column in #{base.to_s} is nullable, change the column to not accept NULL values"
+      unless base.table_exists? && base.columns_hash['deleted'] && !base.columns_hash['deleted'].null
```

Comment:
Since we are here, maybe we should move this `'deleted'` string hardcoded everywhere to a constant and `#freeze` it.